Util.java

```diff
@@ -13,12 +13,24 @@
 	 *
 	 * @param var
 	 *        The floating point number you want to compare
+	 * @param epsilon
+	 *		  The maximum value of var such that it can be considered zero.
 	 * @return
-	 * 		Whether or not it is within Double.MIN_VALUE of zero
+	 * 		Whether or not var is within epsilon of zero
 	 */
 	public static boolean isZero(double var, double epsilon) {
 		return Math.abs(var) < epsilon;
 	}
+
+	/**
+	 * A function for determining if floating point numbers are effectively zero.
+	 * Floating point arithmetic tends to introduce very small errors.
+	 *
+	 * @param var
+	 *        The floating point number you want to compare
```

Comment:
To fix the old again, this would be better phrased as "The floating point number to compare" in both instances